diff --git a/client/index_template.html b/client/index_template.html
index 671ea6f8..098f35a9 100644
--- a/client/index_template.html
+++ b/client/index_template.html
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <title>CELL&times;GENE | Annotate</title>
+    <title>protein&times;location | Annotate</title>
     <style>
       html,
       body,
@@ -37,8 +37,8 @@
   </head>
   <body>
     <script type="text/javascript">
-      window.CELLXGENE = {};
-      window.CELLXGENE.API = {
+      window.PROTEINXLOCATION = {};
+      window.PROTEINXLOCATION.API = {
         prefix: `${location.origin}${location.pathname}api/`,
         version: "v0.2/",
       };
diff --git a/client/package-lock.json b/client/package-lock.json
index 8e78889d..2d22f9a0 100644
--- a/client/package-lock.json
+++ b/client/package-lock.json
@@ -1,11 +1,11 @@
 {
-  "name": "cellxgene",
+  "name": "proteinxlocation",
   "version": "1.3.0",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
-      "name": "cellxgene",
+      "name": "proteinxlocation",
       "version": "1.3.0",
       "license": "MIT",
       "dependencies": {
diff --git a/client/package.json b/client/package.json
index 3973e0ab..3087c9a8 100644
--- a/client/package.json
+++ b/client/package.json
@@ -1,5 +1,5 @@
 {
-  "name": "cellxgene",
+  "name": "proteinxlocation",
   "version": "1.3.0",
   "license": "MIT",
   "description": "cellxgene is a web application for the interactive exploration of single cell sequence data.",
diff --git a/client/src/components/app.js b/client/src/components/app.js
index b91ede88..975d885e 100644
--- a/client/src/components/app.js
+++ b/client/src/components/app.js
@@ -41,7 +41,7 @@ class App extends React.Component {
     const { loading, error, graphRenderCounter } = this.props;
     return (
       <Container>
-        <Helmet title="CELL&times;GENE | Annotate" />
+        <Helmet title="protein&times;location | Annotate" />
         {loading ? (
           <div
             style={{
@@ -51,7 +51,7 @@ class App extends React.Component {
               left: window.innerWidth / 2 - 50,
             }}
           >
-            loading cellxgene
+            loading proteinxlocation
           </div>
         ) : null}
         {error ? (
@@ -63,7 +63,7 @@ class App extends React.Component {
               left: window.innerWidth / 2 - 50,
             }}
           >
-            error loading cellxgene
+            error loading proteinxlocation
           </div>
         ) : null}
         {loading || error ? null : (
diff --git a/client/src/components/framework/logo.js b/client/src/components/framework/logo.js
index 3b7d3881..818a35bb 100644
--- a/client/src/components/framework/logo.js
+++ b/client/src/components/framework/logo.js
@@ -8,8 +8,8 @@ const Logo = (props) => {
       src={icon}
       height={size}
       width={size}
-      alt="CELLxGENE Annotate Logo"
-     />
+      alt="proteinxlocation Annotate Logo"
+    />
   );
 };
 
diff --git a/client/src/components/leftSidebar/infoMenu.js b/client/src/components/leftSidebar/infoMenu.js
index 7d1ee9fa..3e086670 100644
--- a/client/src/components/leftSidebar/infoMenu.js
+++ b/client/src/components/leftSidebar/infoMenu.js
@@ -32,7 +32,7 @@ const InformationMenu = React.memo((props) => {
           <MenuItem
             style={{ cursor: "default" }}
             target="_blank"
-            text={libraryVersions?.cellxgene || null}
+            text={libraryVersions?.proteinxlocation || null}
           />
           <MenuItem style={{ cursor: "default" }} text="MIT License" />
           {tosURL && (
diff --git a/client/src/globals.js b/client/src/globals.js
index 05845540..4eee6fa7 100644
--- a/client/src/globals.js
+++ b/client/src/globals.js
@@ -102,8 +102,12 @@ const CXG_SERVER_PORT =
 
 let _API;
 
-if (typeof window !== "undefined" && window.CELLXGENE && window.CELLXGENE.API) {
-  _API = window.CELLXGENE.API;
+if (
+  typeof window !== "undefined" &&
+  window.PROTEINXLOCATION &&
+  window.PROTEINXLOCATION.API
+) {
+  _API = window.PROTEINXLOCATION.API;
 } else {
   if (CXG_SERVER_PORT === undefined) {
     const errorMessage = "Please set the CXG_SERVER_PORT environment variable.";

